### DISCUSSION TITLE
Quasar v2 JWT and Strapi.

### DISCUSSION THREAD

**Discussion by IvanWala** [quasarframework/quasar#10530] [General - Components / Directives / etc]
Hi 

I'm pretty newbie with Quasar and I need to create auth app with Quasar v2 and Strapi. The problem is I don't find a good resource where I can read how manage the jwt with strapi. I think I need to use the boot files for write "pre conditions" and use in the router instead a middleware like Nuxt.

Some one can post some links for how use Quasar v2 and Strapi for auth.

Thanks A LOT for your help. Really apreciated.

**Comment by ontwikkelfabriek** [comment]
@IvanWala 

I have not seen a Quasar V2 app with a Strapi backend.  Since Strapi supports swagger/openapi you could look up some Vue3 demo's/tutorials with Strapi.

I found this for example:
https://strapi.io/blog/registration-and-login-authentication-with-vue-js-and-strapi
https://strapi.io/blog/how-to-build-and-deploy-a-vue-js-application-using-strapi-on-render

I did find a Qv1 full app/demo ( I would probably take a good look at this one):
https://github.com/very-agency/Quasar-Strapi-Boilerplate

Here are 2 Strapi Qv1( I think) plugins for Quasar:
https://github.com/aysnet1/qv-strapi#readme
https://npm.io/package/@sourcesync/quasar-ui-authorization

**Reply by IvanWala** [reply]
Thank You so much.

I see the Qv1 boilerplate before unfortunately it has many errors in the installation.

I go to try the aysnet1 solution with Quasar v2.

If not I can try the other options. Many thanks.

Edit: aysnet1 solution doesn't work, I go to try Axios with Vuex.

**Comment by suijunqiang** [comment]
I opened a quasar & strapi jwt project here http://skybureau.net:9000/, you may get it from github github.com/suijunqiang
